pkg/policy/identity.go

```diff
@@ -25,12 +25,12 @@ import (
 const (
 	secLabelTimeout = time.Duration(120 * time.Second)
 
-	// All IDs lesser than this value are reserved
+	// MinimalNumericIdentity all IDs lesser than this value are reserved.
```

Comment:
`MinimalNumericIdentity represents the minimal numeric identity not used for reserved purposes.`